capivaraProjects/src/views/Login.tsx

```diff
@@ -68,21 +101,54 @@ class Login extends Component<{}, LoginState>{
       <View style={styles.sectionViewInput}>
         <Text style={styles.sectionText}>E-mail</Text>
         <TextInput
-          style={styles.sectionTextInput}
-          onChangeText={(text) => this.setState({ emailInput: text })}
+          style={ this.inputStyle(this.state.isEmailValid) }
```

Comment:
O método que você está chamando tem acesso ao `this.state`, certo? Nesse sentido, creio que não seja necessário enviar esse parâmetro